handlers.go

```diff
@@ -432,6 +443,130 @@ func (s *Server) handleStats(c echo.Context, u *User) error {
 	return c.JSON(200, out)
 }
 
+// handlePeeringPeersAdd godoc
+// @Summary      Add peers on Peering Service
+// @Description  This endpoint can be used to add a Peer from the Peering Service
+func (s *Server) handlePeeringPeersAdd(c echo.Context) error {
+	var params []peering.PeeringPeer
+	if err := c.Bind(&params); err != nil {
+		return err
+	}
+
+	for _, peerParam := range params {
+
+		var multiAddrs []multiaddr.Multiaddr
+		for _, addr := range peerParam.Addrs {
+			a, err := multiaddr.NewMultiaddr(addr)
+			if err != nil {
+				return &util.HttpError{
+					Code:    400,
+					Message: util.ERR_PEERING_PEERS_ADD_ERROR,
+				}
+			}
+			multiAddrs = append(multiAddrs, a)
+		}
+
+		peerParamId, _ := peer.Decode(peerParam.ID)
+		//	add the peer(s)
+		s.Node.Peering.AddPeer(
+			peer.AddrInfo{
+				peerParamId,
+				multiAddrs,
+			})
+	}
+	return c.JSON(200, params)
+}
+
+// handlePeeringPeersRemove godoc
+// @Summary      Remove peers on Peering Service
+// @Description  This endpoint can be used to remove a Peer from the Peering Service
+func (s *Server) handlePeeringPeersRemove(c echo.Context) error {
+	var params []peer.ID
+
+	type PeeringPeerRemoveMessage struct {
+		Message     string    `json: Message`
+		PeersRemove []peer.ID `json: Peers`
+	}
+
+	if err := c.Bind(&params); err != nil {
+		return &util.HttpError{
+			Code:    400,
+			Message: util.ERR_PEERING_PEERS_REMOVE_ERROR,
+		}
+	}
+
+	for _, peerId := range params {
+		s.Node.Peering.RemovePeer(peerId)
+	}
+
+	return c.JSON(200, PeeringPeerRemoveMessage{"Removed the following Peers from Peering", params})
```

Comment:
Use `http.StatusOK` or, better yet, `http.StatusAccepted` instead of `200`. Also do this to all the other returns on the endpoints

Reply:
Ok that's a good suggestion. I was following the same response from other endpoints.